src/GeositeFramework/js/Plugin.js

```diff
@@ -196,6 +196,14 @@ require(['use!Geosite',
                 } else {
                     this.get('pluginObject').deactivate();
                     this.trigger('plugin:deselected');
+                    // Remove the `.nav-apps-narow` class if the
+                    // de-selecting the plugin has left no plugins
+                    // active.
+                    if ($('.sidebar').filter(function() {
```

Comment:
I don't think we should be using the CSS properties for state in the app.  Instead, you can check if all plugins are  not `active`:
```js
if (this.collection.all({active:false})) {
    $('.nav-apps').removeClass('nav-apps-narrow');
}
````